eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -287,6 +284,25 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <module name="MissingJavadocType"/>
     <module name="MissingJavadocPackage"/>
 
+    <module name="EmptyCatchBlock" />
+
+  <!-- Bug in comments indentation - https://github.com/checkstyle/checkstyle/issues/9105 -->
+  <!-- <module name="CommentsIndentation" />-->
+
+    <module name="SimplifyBooleanReturn"/>
+
+    <!--  The parts of a class or interface declaration should appear in the following order:
+      1. Class (static) variables. First the public class variables, then protected, then package level (no access modifier), and then private.
+      2. Instance variables. First the public class variables, then protected, then package level (no access modifier), and then private.
+      3. Constructors
+      4. Methods -->
+    <module name="DeclarationOrder"/>
```

Comment:
Is there any configuration options for this check? I'm wondering if we could start with a simplified version where the separate code types, class variables, instance variables, constructors, and methods, need to be group without the more intricate scoping orders.

Reply:
Yeah I beleive we could skip/ignore access modifiers validation. But still keep the ordering in place with this
 ```xml
<module name="DeclarationOrder">
                 <property name="ignoreModifiers" value="false" />
         </module>
```